### PR TITLE
Add controller

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary
Add controller

## Screenshots
GIF

## JIRA Card
https://trello.com/c/ZLUttS4Z/17-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

**Comment by darkBasic666** [comment]
It's a good moment to read a [conventions and good practices](https://docs.google.com/document/d/1xBoy41sG9vkrVsOd109OQ61LBQxPzZ2_F3K9e5jfHQY/edit).

**Comment by darkBasic666** [comment]
You should write commits messages in English and group them into general messages.

**Comment by darkBasic666** [comment]
Try to don't commit for little changes.

**Review by darkBasic666 — APPROVED** [review]

File: training/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/dbname
-spring.datasource.username= POSTGRESUSERNAME
-spring.datasource.password= POSTGRESPASSWORD
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username= postgres
+spring.datasource.password= hola1234
```
Comment (rabinovichm) [inline training/src/main/resources/application.properties:3]: In the real-life, you should never leave credentials exposed in the code. You can use the _environment variables_ of IntelliJ to set these values.
You should go to `Run -> Edit Configurations -> Environmnet Variables` and paste them exactly as you have them in the application.properties separated by `;`
For example: `spring.datasource.username=sampleUsername;spring.datasource.password=samplePass`

**Review by rabinovichm — APPROVED** [review]

**Pull request merged**